### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by barunio**
It would be particularly helpful to review the README and make sure it makes sense.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,91 @@
...
+To seed your database with a couple data fields, you would just include the
+following in your seeds.rb file:
+
+```ruby
+seeds = [
+  { data_type: "Oil", name: "btu", description: "Oil usage" },
+  { data_type: "Water", name: "gallons", description: "Water usage" }
+]
+
+Seeder.create seeds, [:data_type, :name], DataField
```
Comment (jdewyea) [inline README.md]: Does `:description` need to be included in the array?

Comment (jdewyea) [inline README.md]: Ah, nevermind. I was misreading the `create` method.

Comment (barunio) [inline README.md]: Well, the fact that it was confusing means the README isn't as clear as it should be. One ought to be able to understand this without needing to look at the code. Any suggestions on how to modify this doc to make it clearer?

Comment (jdewyea) [inline README.md]: I might add a line explaining that the second argument it takes is an array of one or more column names to use as the primary key for identifying existing records. That should be sufficient, I think.

File: lib/seeder.rb [hunk 2]
```diff
@@ -1,78 +1,66 @@
...
-    }
+  def self.create(data, keys, model)
+    new(data, keys, model).create
+  end
 
-    def existing_data_keys(ar_model)
-      keys = @primary_keys[ar_model.name]
-      ar_model.all.map{|item| item.attributes.symbolize_keys.values_at(*keys) }
-    end
+  def initialize(data, keys, model)
```
Comment (jdewyea) [inline lib/seeder.rb:8]: Another minor suggestion -- I think from what I've seen in the rails source code the tendency is to use `model` to refer to an ActiveRecord instance, and use `model_class` to refer to the class itself. Not sure if it's worth changing that here, but I figured I'd mention it.

Comment (barunio) [inline lib/seeder.rb:8]: Hm, I haven't seen any relevant code in the Rails source, but I'm going to keep this as is. I think the term `record` makes more sense for an ActiveRecord instance, and `model` is a good term for the model object, which is the class that a record is an instance of. `model_class` almost seems more like the parent class of the model object to me.

**Comment by barunio** [comment]
@jdewyea I added a commit that clarifies the README a bit.

**Comment by jdewyea** [comment]
:thumbsup:

**Comment by fixlr** [comment]
:thumbsup:

**Comment by barunio** [comment]
Huh, I'd completely forgotten about this PR :)

**Comment by barunio** [comment]
:rabbit: :rabbit2:

**Pull request merged**